### PR TITLE
fix: block height increment in sdk mine_block

### PR DESCRIPTION
**Pull request by hugocaillard**
### Description

In `sdk.mine_block`, `session.advance_chain_tip(1);` must be called before executing the contract calls.

This PR includes other minor fix

### CODE REVIEW

File: components/clarinet-sdk-wasm/src/core.rs [hunk 1]
```diff
@@ -854,6 +855,11 @@ impl SDK {
             .into_serde()
             .map_err(|e| format!("Failed to parse js txs: {:}", e))?;
 
+        {
```
Comment (hugocaillard) [inline components/clarinet-sdk-wasm/src/core.rs:858]: actual fix here

File: components/clarinet-sdk-wasm/src/core.rs [hunk 2]
```diff
@@ -854,6 +855,11 @@ impl SDK {
...
+            let session = self.get_session_mut();
+            session.advance_chain_tip(1);
+        }
```
Comment (hugocaillard) [inline components/clarinet-sdk-wasm/src/core.rs:861]: Actual fix here

File: components/clarity-lsp/src/common/requests/signature_help.rs [hunk 3]
```diff
@@ -23,7 +23,7 @@ pub fn get_signatures(
     ]
     .contains(&function_name.as_str())
     {
-        // showing signature help for define-<function>, define-trait, let and bug adds to much noise
+        // showing signature help for define-<function>, define-trait, let, and begin adds to much noise
```
Comment (hugocaillard) [inline components/clarity-lsp/src/common/requests/signature_help.rs:26]: typo fix

**Comment by hugocaillard** [comment]
Thanks @BowTiedRadone for reporting this issue

**Review by MicaiahReid — APPROVED** [review]

**Pull request merged**